docs/dev/table/hive/read_write_hive.md

```diff
@@ -111,16 +111,44 @@ __________ __________
 
 ## Writing To Hive
 
-Similarly, data can be written into hive using an `INSERT INTO` clause. 
+Similarly, data can be written into hive using an `INSERT` clause. Consider there is a mytable table with two columns: name, age.
 
 {% highlight bash %}
-Flink SQL> INSERT INTO mytable (name, value) VALUES ('Tom', 4.72);
+# ------ Insert with append mode ------ 
+Flink SQL> INSERT INTO mytable SELECT 'Tom', 25;
+
+# ------ Insert with overwrite mode ------ 
+Flink SQL> INSERT OVERWRITE mytable SELECT 'Tom', 25;
```

Comment:
are "append mode" and "overwrite mode" official names? 

maybe just say "insert new values by appending to existing table" and "insert new values by overwriting the whole existing table"

Reply:
Will modify to:
```
INSERT INTO will append to the table or partition, keeping the existing data intact.
INSERT OVERWRITE will overwrite any existing data in the table or partition.
```